src/pocapi.cpp

```diff
@@ -55,8 +56,8 @@ OVMS_Status* OVMS_ServerStartFromConfigurationFile(OVMS_Server* server,
     ovms::ServerImpl* srv = (ovms::ServerImpl*)server;
     ovms::GeneralOptionsImpl* go = (ovms::GeneralOptionsImpl*)general_options;
     ovms::MultiModelOptionsImpl* mmo = (ovms::MultiModelOptionsImpl*)multi_model_specific_options;
-    srv->start(go, mmo);
-    return 0;
+    std::int64_t res = srv->start(go, mmo);
```

Comment:
OVMS_Status is fully fledged class, not pure StatusCode. At least leave TODO

Reply:
Added todo